roles/openshift_metrics/tasks/cleanup.yaml

```diff
@@ -17,3 +17,14 @@
     {{ openshift.common.admin_binary }} policy remove-cluster-role-from-user
     cluster-reader 'system:serviceaccount:{{ project }}:heapster'
   when: "'system:serviceaccount:{{ project }}:heapster' in ret.stdout"
+- name: test view permissions for the hawkular service account
+  command: >
+    {{ openshift.common.client_binary }} get rolebindings
+    -o jsonpath='{.items[?(@.roleRef.name == "view")].userNames}'
+  register: ret
```

Comment:
Scope of variables is still a little bit of a mystery to me so, does this need to be something more descriptive and unique?